### DISCUSSION TITLE
Could not resolve host: flex.symfony.com

### DISCUSSION THREAD

**Discussion by chriscroome** [kimai/kimai#3518] [On-Premise: Installation & Updates]
Updating a Kimi install to version 1.24.0 failed today:

```bash
git fetch --tags
git checkout 1.24.0
composer install --no-dev --optimize-autoloader -n

Installing dependencies from lock file
Verifying lock file contents can be installed on current platform.
The following exception probably indicates you have misconfigured DNS resolver(s)

In CurlDownloader.php line 375:
                                                                                                                   
  curl error 6 while downloading https://flex.symfony.com/versions.json: Could not resolve host: flex.symfony.com  
                                                                                                                   

install [--prefer-source] [--prefer-dist] [--prefer-install PREFER-INSTALL] [--dry-run] [--dev] [--no-suggest] [--no-dev] [--no-autoloader] [--no-progress] [--no-install] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--apcu-autoloader-prefix APCU-AUTOLOADER-PREFIX] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--] [<packages>...]
```

The DNS servers for `symfony.com` are:

```bash
whois symfony.com | grep ^Name
Name Server: JOSH.NS.CLOUDFLARE.COM
Name Server: NADIA.NS.CLOUDFLARE.COM
```

And querying them about the `flex.symfony.com` domain:

```bash
dig @JOSH.NS.CLOUDFLARE.COM flex.symfony.com +short
dig @NADIA.NS.CLOUDFLARE.COM flex.symfony.com +short
```

Returns no results -- the sub-domain `flex.symfony.com` does not exist, this is because as of May 2022 [the old Flex infrastructure is shutting down](https://symfony.com/blog/the-old-flex-infrastructure-is-shutting-down) so to [work around](https://symfony.com/blog/upgrade-flex-on-your-symfony-projects) this:

```bash
composer update symfony/flex --no-plugins --no-scripts
```

**Comment by kevinpapst** [comment]
From which version did you update?

If you haven't updated Kimai in a while, that might happen because you have an old flex installed that is still trying to call this URL. Sorry, but there is nothing that I can do about it.

But thanks for sharing the solution 👍 

I will move this to a discussion, as it might be helping others.

**Comment by chriscroome** [comment]
Thanks @kevinpapst I believe it was from version `1.12` so my bad for not updating for ages... :roll_eyes:

**Reply by kevinpapst** [reply]
Yep, that is the reason. Version 1.12 was released on 14 Dec 2020, long before Symfony announced to shut down the Flex server 🤷

**Reply by kevinpapst** [reply]
Ha, today I updated a customer from 1.14 and had the same issue. Thanks again for sharing the solution 😁
